website/docs/guides/fusion-upgrade.md

```diff
@@ -0,0 +1,432 @@
+
+### Step 4: Run an immediate test job
+
+Validate the upgrade by running a job:
+
+1. From the **Environments** page, click on your production environment.
+2. Select a critical job that covers a good subset of your models.
+3. Click **Run now** to execute the job immediately.
+4. Monitor the job run closely:
+   - Check the **parse** and **compile** steps.
+   - Verify all models build successfully.
+   - Confirm tests pass as expected.
+   - Review the logs for any unexpected warnings.
+
+If the job succeeds, your production upgrade is successful!
+
+### Step 5: Enable state-aware orchestration (optional but recommended)
+
+One of <Constant name="fusion" />'s most powerful features is [state-aware orchestration](/docs/deploy/state-aware-about), which automatically determines which models need rebuilding based on code or data changes. This can reduce warehouse costs by 30% or more.
+
+New jobs automatically have state-aware orchestration enabled in <Constant name="fusion" /> environments.
+
+To enable it for existing jobs:
+
+1. Navigate to **Deploy** → **Jobs**.
+2. Click on a production job to open its settings.
+3. Click **Edit** in the top right.
+4. Scroll to **Execution settings**.
+5. Check the box for **Enable Fusion cost optimization features**.
+6. Expand **More options** to see additional settings:
+   - **State-aware orchestration**
+   - **Efficient testing**
+7. Click **Save**.
+
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/example-triggers-section.png" width="90%" title="Enable Fusion cost optimization features"/>
+
+Repeat this for all production jobs to maximize cost savings. For more details, see [Setting up state-aware orchestration](/docs/deploy/state-aware-setup).
+
+### Step 6: Monitor production jobs
+
+Watch your production jobs closely for the first 24-48 hours:
+
+- **Check scheduled job runs:** Navigate to **Deploy** → **Jobs** → **Run history**
+- **Monitor run times:** Compare to historical averages. You should see significant improvements.
+- **Review the state-aware interface**: Check the [Models built and reused chart](/docs/deploy/state-aware-interface) to see cost savings in action.
+- **Watch for warnings**: Review logs for any unexpected messages.
+
+:::tip State-aware monitoring
+
+With state-aware orchestration enabled, you'll see models marked as **Reused** in the job logs when they don't need rebuilding. This is expected behavior and indicates cost savings!
+
+:::
+
+### Step 7: Validate downstream integrations
+
+Ensure all systems dependent on your production data still function correctly:
+
+1. **BI tools:** Verify dashboards and reports refresh properly.
+2. **Data consumers:** Confirm downstream teams can access and query data.
+3. **APIs and integrations:** Test any applications that consume dbt outputs.
+4. **Semantic Layer:** If using the [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl), verify metrics queries work.
```

Comment:
Are the metrics queries dependent on still running a dbt Core job in the background or anything? Or is it just development that is pinned to Core?